src/runtime/lpython_builtin.py

```diff
@@ -1085,6 +1085,65 @@ def _lpython_str_isspace(s: str) -> bool:
             return False
     return True
 
+@overload
+def _lpython_str_center(s: str, width: i32, fillchar: str) -> str:
+    """
+    Return centered in a string of length width. 
+    Padding is done using the specified fillchar (default is an ASCII space). 
+    The original string is returned if width is less than or equal to len(s).
+    """
+    if(len(fillchar) != 1):
+        raise TypeError("The fill character must be exactly one character long")
+    if(width <= len(s)):
+        return s
+    width -= len(s)
```

Comment:
```suggestion
    str_len: i32 = len(s)
    if width <= str_len:
        return s
    width -= str_len
```

Although `len(x)` is a O(1) operation, let's have only a single function call.

Reply:
@farah-salama do you mind updating this?

Reply:
Updated.